app/src/main/res/layout-sw600dp-port/onboarding_slide_final.xml

```diff
@@ -47,37 +46,36 @@
       android:id="@+id/slide_description_text_view"
       android:layout_width="0dp"
       android:layout_height="wrap_content"
-      android:layout_marginStart="48dp"
-      android:layout_marginTop="16dp"
-      android:layout_marginEnd="48dp"
+      android:layout_marginStart="192dp"
+      android:layout_marginTop="24dp"
+      android:layout_marginEnd="192dp"
       android:fontFamily="sans-serif"
       android:gravity="center_horizontal"
       android:maxLines="2"
       android:minLines="2"
       android:text="@string/onboarding_slide_3_description"
       android:textColor="@color/oppiaPrimaryText"
-      android:textSize="16sp"
+      android:textSize="24sp"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toBottomOf="@id/slide_title_text_view" />
 
     <Button
       android:id="@+id/get_started_button"
       android:layout_width="match_parent"
-      android:layout_height="wrap_content"
-      android:layout_marginStart="56dp"
-      android:layout_marginEnd="56dp"
-      android:layout_marginTop="36dp"
-      android:layout_marginBottom="32dp"
+      android:layout_height="54dp"
```

Comment:
Do not keep fixed button height. Instead just use `minHeight="48dp"` and this will make sure that it works well on each screen.

Reply:
Done